monitoring/definitions/frontend.go

```diff
@@ -745,6 +746,138 @@ func Frontend() *monitoring.Container {
 					},
 				},
 			},
+			{
+				Title:  "Sentinel queries (only on sourcegraph.com)",
+				Hidden: true,
+				Rows: []monitoring.Row{
+					{
+						{
+							Name:        "mean_successful_sentinel_duration_5m",
+							Description: "mean successful sentinel search duration over 5m",
+							Query:       `sum(rate(src_search_response_latency_seconds_sum{source=~"searchblitz.*", status="success"}[5m])) / sum(rate(src_search_response_latency_seconds_count{source=~"searchblitz.*", status="success"}[5m]))`,
+							Warning:     monitoring.Alert().GreaterOrEqual(5, nil).For(10 * time.Minute),
+							Critical:    monitoring.Alert().GreaterOrEqual(8, nil).For(15 * time.Minute),
+							Panel:       monitoring.Panel().LegendFormat("duration").Unit(monitoring.Seconds).With(noLegend()),
+							Owner:       monitoring.ObservableOwnerSearch,
+							PossibleSolutions: `
+							    - Look at the breakdown by query to determine if a specific query type is being affected
+								- Check for high CPU usage on zoekt-webserver
+								- Check Honeycomb for unusual activity`,
```

Comment:
```suggestion
								- Check Honeycomb for unusual activity
							`,
```

All these lines need to have the ending on a separate line like this I think